packages/cozy-ccc-libs/src/utils.js

```diff
@@ -23,83 +14,4 @@ async function blobToBase64(blob) {
   return reader.result
 }
 
-export { kyScraper, blobToBase64 }
-
-function addKyScraper(_request, _options, response) {
-  response.$ = async () => Cheerio.load(await response.text())
-  response.scrape = async (...args) => {
-    if (typeof args[0] !== 'string') {
-      args.unshift(await response.$())
-    }
-    return scrape(...args)
-  }
-  response.getFormData = async formSelector =>
-    parseForm(await response.$(), formSelector)
-  return response
-}
-
-function parseForm($, formSelector) {
-  const form = $(formSelector).first()
-
-  if (!form.is('form')) {
-    throw new Error('INVALID_FORM')
-  }
-
-  const inputs = {}
-  const arr = form.serializeArray()
-  for (let input of arr) {
-    inputs[input.name] = input.value
-  }
-  return inputs
-}
-
-function scrape($, specs, childSelector) {
-  // Only one value shorthand
-  if (
-    typeof specs === 'string' ||
-    (specs.sel && typeof specs.sel === 'string')
-  ) {
-    const { val } = scrape($, { val: specs })
-    return val
-  }
-
-  // Several items shorthand
-  if (childSelector !== undefined) {
-    return Array.from(($.find || $)(childSelector)).map(e =>
-      scrape($(e), specs)
-    )
-  }
-
-  // Several properties "normal" case
-  const res = {}
-  Object.keys(specs).forEach(specName => {
-    const spec = mkSpec(specs[specName])
-    let data = spec.sel ? $.find(spec.sel) : $
-    if (spec.index) {
-      data = data.get(spec.index)
-    }
-    let val
-    if (spec.fn) {
-      val = spec.fn(data)
-    } else if (spec.attr) {
-      val = data.attr(spec.attr)
-    } else {
-      val = data
-      val = val && val.text()
-      val = val && val.trim()
-    }
-    if (spec.parse) {
-      val = spec.parse(val)
-    }
-    res[specName] = val
-  })
-  return res
-}
-
-function mkSpec(spec) {
-  if (typeof spec === 'string') {
-    return { sel: spec }
-  } else {
-    return spec
-  }
-}
```

Comment:
Are we sure to want to create BC right now? If the lib is well done, this part will be tree-shaked and will not be imported to the bundle if the konnector doesn't use it.

Reply:
Puisque l'utilisation de cheerio est un mode de fonctionnement qu'on veut éviter et vu qu'on est de toutes façons obligés de modifier tous les connecteurs ccc juste à la suite, ça me semblait être un bon moment pour le faire oui. Sachant que pour surmonter cette modification, il suffit de récupérer ce code dans le connecteur amazon qui est le seul utilisant cette fonction.

Reply:
> Puisque l'utilisation de cheerio est un mode de fonctionnement qu'on veut éviter

C'est vraiment pas dit. En avez-vous discuté avec @Ldoppea avant de prendre cette décision ?

Reply:
Non, on n'en a pas parlé avec @Ldoppea et ça va être compliqué cette semaine. J'en parlerait avec lui dès que possible et je pourrais toujours réintroduire cette fonctionnalité si nécessaire.